test/controllers/AssessmentControllerSpec.scala

```diff
@@ -166,7 +166,8 @@ class AssessmentControllerSpec extends SpecBase with MockitoSugar with BeforeAnd
                 .url
 
               withClue("must cleanse the user answers data") {
-                verify(mockSessionRepository).clearData(eqTo(emptyUserAnswers.id), eqTo(CategorisationJourney))
+                verify(mockSessionRepository, times(1))
```

Comment:
just a style choice but why are you adding times(1) to all the verifies? The default is times(1) already so you don't need to add it in. If you check our code historically we were using times(1) and then we removed them - I think it was at Mauro's suggestion.